crates/ruff_linter/src/rules/ruff/mod.rs

```diff
@@ -429,6 +429,7 @@ mod tests {
     #[test_case(Rule::DataclassEnum, Path::new("RUF049.py"))]
     #[test_case(Rule::StarmapZip, Path::new("RUF058_0.py"))]
     #[test_case(Rule::StarmapZip, Path::new("RUF058_1.py"))]
+    #[test_case(Rule::OperationOnClosedIO, Path::new("RUF061.py"))]
```

Comment:
We can move the test to the regular `rules` test because the rule contains no `settings.preview_mode` checks.